src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralTest.kt

```diff
@@ -38,35 +39,41 @@ class DraftReferralTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `create draft referral dto from referral`() {
-    val createdDate = OffsetDateTime.parse("2020-12-04T10:42:43+00:00")
-    val id = UUID.fromString("70d3a47c-d539-4f76-8fc9-1c50e34aea29")
-    val draftReferral = DraftReferral(Referral(id = id, created = createdDate))
-
-    assertThat(draftReferral.id).isEqualTo("70d3a47c-d539-4f76-8fc9-1c50e34aea29")
-    assertThat(draftReferral.created).isEqualTo("2020-12-04T10:42:43Z")
+  fun `test serialization of newly created referral`() {
+    val referral = Referral(
+      id = UUID.fromString("3B9ED289-8412-41A9-8291-45E33E60276C"),
+      created = OffsetDateTime.parse("2020-12-04T10:42:43+00:00")
+    )
+    val out = json.write(DraftReferral(referral))
+    assertThat(out).isEqualToJson(
+      """
+      {"id": "3b9ed289-8412-41a9-8291-45e33e60276c", "created": "2020-12-04T10:42:43Z"}
+    """
+    )
   }
 
   @Test
-  fun `determine json for draft referral dto`() {
-    val uuid = "70d3a47c-d539-4f76-8fc9-1c50e34aea29"
-    val draftReferralJson =
-      """{
-        "id":"$uuid",
-        "created":"2020-12-04T10:42:43Z"
-      }"""
-
-    val mapper = jacksonObjectMapper()
-
-    val draftReferral = DraftReferral(
-      Referral(
-        id = UUID.fromString(uuid),
-        created = OffsetDateTime.parse("2020-12-04T10:42:43+00:00"),
-        completionDeadline = LocalDate.parse("2021-10-05")
-      )
+  fun `test serialization of referral with completionDeadline`() {
+    val referral = Referral(
+      id = UUID.fromString("3B9ED289-8412-41A9-8291-45E33E60276C"),
+      created = OffsetDateTime.parse("2020-12-04T10:42:43+00:00"),
+      completionDeadline = LocalDate.of(2021, 2, 12)
+    )
+    val out = json.write(DraftReferral(referral))
+    assertThat(out).isEqualToJson(
+      """
+      {"id": "3b9ed289-8412-41a9-8291-45e33e60276c", "created": "2020-12-04T10:42:43Z", "completionDeadline": "2021-02-12"}
```

Comment:
_further question, outside this PR_
Since we're in winter, we don't currently have an offset from UTC. Would this still serialise to UTC during summer dates?